internal/services/disbursement_management_service.go

```diff
@@ -210,19 +220,88 @@ func (s *DisbursementManagementService) StartDisbursement(ctx context.Context, d
 			}
 		}
 
-		// 4. Update all correct payment status to `ready`
+		// 4. Check if there is enough balance from the distribution wallet for this disbursement along with any pending disbursements
+		rootAccount, err := s.horizonClient.AccountDetail(
+			horizonclient.AccountRequest{AccountID: distributionPubKey})
+		if err != nil {
+			return fmt.Errorf("error getting details for root account from horizon client: %w", err)
```

Comment:
In Go, error we don't start error messages with the `error` word. Otherwise, the stack trace would look like `error doing A: error doing B, error doing C: ...`m which is redundant.

The libraries usually don't start the error messages with `error` either.

I know there are some places in the code that are not following this pattern thoroughly though, so don't worry about changing it in other places for now.

Reply:
good point, I've noticed this as well. the nesting of the errors does feel very redundant